### ISSUE TITLE
Where the strange resolution of 496x384 came from.

### ISSUE THREAD

**Issue by BackPonBeauty** [BackPonBeauty/Supermodel3-PonMi#1]
スパイクアウトFEの画面比率がアーケード筐体で一般的な4:3ではなく、縦長に見える点に対して、強い違和感を覚えていました。

通常、日本のアーケード筐体に装備されているジョイスティックは、掴む部分が真円のボールで、縦長の形状ではありません。それにも関わらず、スパイクアウトFEのアドバタイズ画面を見ると、映像が縦長で、掴む部分も縦長に見えています。

I felt a strong sense of discomfort with the fact that the screen ratio of Spike Out FE is not the usual 4:3 for arcade cabinets, but appears vertically elongated.

Normally, the joysticks equipped in Japanese arcade cabinets have a perfectly round ball-shaped grip, and are not vertically elongated. Despite this, when I look at the advertisement screen for Spike Out FE that I captured, the image is vertically elongated, and the gripping part also appears to be vertically elongated.
![111](https://github.com/user-attachments/assets/a8ce5440-17cc-4f85-84c3-13af48b072a7)

この現象の理由を調査した結果、スパイクアウトFEが動作している[Model3](https://ja.wikipedia.org/wiki/MODEL3)アーケードシステムのグラフィックスには、ロッキード・マーティンのReal3D/Pro-1000が使用されていることがわかりました。[資料](https://retrocdn.net/images/7/75/Real3DPro1000ProductDescription.pdf)によれば、このグラフィックスカードの最低解像度は640x480であり、通常は60Hzで動作します。しかし、496x384という奇妙な解像度がどこから発生したのかは、私には不明です。

また、当時のセガのアーケード筐体、例えばアストロシティやブラストシティに使用されていたモニターは、ナナオ製やサンワ製の29インチで、16セクター×12セクターの4:3の画面比率を持つものでした。これが、一般的なアーケード環境での基準となっています。

**この調査を基に、今回は640x480の4:3の解像度で、正確な出力設定に変更を加えました。**

しかし、申し訳ありませんが、この修正はまだリリースしません。理由は、496x384という数値の出所が不明であり、これが公式の意図によるものかどうか確認を取る必要があるためです。もし、この解像度が正式で、4:3に戻すことが適切でないのであれば、この修正を公開するべきではないと考えています。

もしかしたら、当時の日本の雑誌から誤った情報が広まったのかもしれません。彼らは時に写植ミスをすることがあります。「ハンドルを右に」と書くところを「インド人を右に」と誤植したのは有名な話で、日本ではネットミームになっています。

結論として、この問題の完全解決にはもう少し時間が必要ですが、解決に向けて着実に進んでいます。

「リンダの尻はもっと大きいはずだ！」（ポン美）

After investigating the reason for this phenomenon, I found that the graphics card of the [Model3 arcade system](https://ja.wikipedia.org/wiki/MODEL3) on which Spikeout FE runs uses Lockheed Martin's Real3D/Pro-1000. According to the [documentation](https://retrocdn.net/images/7/75/Real3DPro1000ProductDescription.pdf), the minimum resolution for this graphics card is 640x480, and it usually runs at 60Hz. However, I don't know where the strange resolution of 496x384 came from.

Also, the monitors used for Sega's arcade cabinets at the time, such as Astro City and Blast City, were 29 inches made by Nanao and Sanwa, with a 4:3 screen ratio of 16 sectors x 12 sectors. This is the standard in general arcade environments.

**Based on this investigation, we have made changes to the exact output settings, this time with a 4:3 resolution of 640x480.**

However, I'm sorry to say that we will not release this fix yet. The reason is that the source of the number 496x384 is unknown and we need to confirm whether this is the official intention. If this resolution is official and changing it back to 4:3 is not appropriate, I think this fix should not be released.

Maybe the wrong information was spread by a Japanese magazine at the time. They sometimes make typographical mistakes. The typo of "Turn the steering wheel to the right" as "Turn the Indian to the right" is a famous story in Japan and has become an internet meme.

In conclusion, it will take some time to completely solve this problem, but we are steadily moving towards a solution.

"Linda's butt is bigger," is what I wanted to tell everyone. (Ponmi)
![222](https://github.com/user-attachments/assets/bd9ea34b-654c-46df-901d-9c45f585dd9e)
![333](https://github.com/user-attachments/assets/0b2b50b5-ec2a-4a15-8dc6-f1ff9f39df8e)

**Comment by BackPonBeauty** [comment]
It has become clear that a 496x384 resolution—while not a 4:3 aspect ratio—is indeed possible and used by various titles. Many games did not output at a perfect 4:3 resolution, but instead were adjusted to appear as 4:3 on arcade monitors.



For example, the game 'Tenchi wo Kurau II - sekiheki no tatakai' (wofj) has a 384x224 resolution, which is wider than 4:3, but adjusting it to display in a 4:3 aspect ratio replicates the original arcade experience.
![tenchi](https://github.com/user-attachments/assets/c8deccc1-705b-4b53-8acd-2e580635ebab)

Regarding Spikeout FE, setting a 4:3 resolution with the Stretch option enabled results in an almost 4:3 aspect ratio, as expected. However, enabling both Widescreen and Stretch in a 16:9 resolution leads only the 4:3 portion to stretch, creating an inaccurate display.
![0000](https://github.com/user-attachments/assets/375fca35-2d6c-44d6-9dc7-1738459d9b24)
![0002](https://github.com/user-attachments/assets/a262ce90-7984-4141-aa2a-0e6edd50048b)

When modifying the base resolution to 640x480 (4:3), the FOV (Field of View) appears slightly different compared to the original, which is likely due to these resolution discrepancies affecting the rendering output.

![0001](https://github.com/user-attachments/assets/a309f4c2-c657-4185-ad36-319edb4aeeab)
![0003](https://github.com/user-attachments/assets/c850e717-8497-437a-99dd-dfc0215d7a35)
![123](https://github.com/user-attachments/assets/d99b133b-d092-4c31-bf3e-8453df9d042f)

The difference in FOV is slight, but the controllability of the character is quite different.

Which one do you prefer?
I'd love to hear your opinions.

PonMi(ニーコニーコブーン⊂二二二（ ＾ω＾）二二⊃ﾌﾞｰﾝ)

**Comment by BackPonBeauty** [comment]
![image](https://github.com/user-attachments/assets/9a3e3fd4-03e2-4ae0-9d94-f6676bdac434)
ポン美☆オラオラオラ～

**Comment by WarpedPolygon** [comment]
Hi PonMi,

This is awesome. Sorry I overlooked it. The True AR mode/fix you have included is much better. I felt the other aspect ratio was a bit skinny myself.

I am now moving over to your build of Supermodel for my primary. I have soo many questions but don't want to pester you, however, I do wanna put out a video on the PonMi Edition. Let me know if you catch a spare 10 minutes

**Comment by BackPonBeauty** [comment]
Hello WarpedPolygon

In the latest version, True AR mode is 512x384.

Videos are welcome. Please follow YouTube policies.

If you have any questions, feel free to post them here or send me a private message on Discord.

Get a glimpse into my hobby.
[Discord ポン美](https://discord.gg/8SF2fKm6f8)

Discord for sharing and playing SpikeoutFE on Parsec
[Discord PonMi](https://discord.gg/f7Wc6DwUDG)
